app/src/main/res/values/strings.xml

```diff
@@ -3,17 +3,17 @@
     <string name="action_search">Search</string>
     <string name="action_add">Add</string>
     <plurals name="selectedCardCount">
-        <item quantity="one"><xliff:g>%d</xliff:g> card selected</item>
-        <item quantity="other"><xliff:g>%d</xliff:g> cards selected</item>
+        <item quantity="one"><xliff:g>%d</xliff:g> selected</item>
+        <item quantity="other"><xliff:g>%d</xliff:g> selected</item>
     </plurals>
     <string name="noGiftCards">Click the + plus button to add a card, or import some from the ⋮ menu first.</string>
-    <string name="noGiftCardsGroup">You don\'t have any loyalty cards yet. Once you\'ve added some you can assign them to the group here.</string>
-    <string name="noMatchingGiftCards">Didn\'t find anything. Try changing your search.</string>
+    <string name="noGiftCardsGroup">If you add some cards you can assign them to this group afterwards.</string>
```

Comment:
I'm not a fan of changing this from "here" to "afterwards", because this appears exactly at the UI location where you can choose which cards belong to a group. The old string is very explicitly: "once you have created cards, come back to this screen to add them to this group". The new one is vague "if you do X you can do Y after that".